src/features/revurdering/sharedMessages-nb.ts

```diff
@@ -23,6 +23,10 @@ export default {
     'revurdering.ingenEndring':
         'Ny beregning fører til mindre enn 10 % endring av utbetalingen. Vi lagrer beregningen, men utbetalingen til bruker blir ikke endret.',
 
+    'revurdering.ingenEndring.gregulering.tittel': 'G-regulering',
+    'revurdering.ingenEndring.gregulering.beskrivelse':
+        'Denne revurderingen har samme utbetalingsbeløp per måned. Det er derfor ingen simulering. Vi lagrer den nye informasjonen som er satt in.',
```

Comment:
Kan ta screenshot av hvordan dette ser ut i nettleseren og legge i trello-kortet, så blir det enklere for fag å akseptanseteste